### DISCUSSION TITLE
Proposal for an age rated system of tagging

### DISCUSSION THREAD

**Discussion by Ribenaberri** [Glimesh/company#19] [Ideas]
Myself and Ember were discussing the use of the 18+ tag within Glimesh and feel there is a strong argument for a 3-tag system for age rating. 

My reasons are the following: 

18+, as defined by our TOS encompasses the following: 

Channels that would contain mature content must be marked with an 18+ content tag. Mature content is anything in these categories: 

* Body Art that requires the artist to be undressed (Please see body art section in Rules of Conduct)
* Sleeping Streams
* Games containing brief nudity

You must be 18+ years of age to stream these categories. By using these tags you are agreeing that you are over 18 years old.

This, by very definition, as things stand, means anything else outside of this would be open to all ages. 

My question is, is this appropriate?

Is it appropriate that a stream full of swearing, or blood and gore, should be labelled an all ages stream? There are bound to be people out there, and parents of the younger users, who would be shocked to see this content freely available without label or warning as to what the content is. 

My thought is that we perhaps introduce a mandatory selection in the tags. Age Rating. 

1. All Ages. This content is safe for anyone to view, no risk here! 
2. Teen (or something to that effect). There is the potential for blood and gore, swearing etc. 
3. 18+, as defined by our TOS. 

My argument for NOT making the 18+ stronger is that too do so may see those who swear in their games subject to an 18+ rating, which would mean streamers such as Call Of Duty players, who, if you base it on game rating (which im aware the platform isnt about, but for the sake of how a parent may make an informed decision about games for their children to play) is 16+, wouldn't necessairly be able to stream it. 

I am sure there will be many streamers out there who are mindful of their content, and may feel uncomfortable labelling their content all ages, but to label it 18+ would impact them heavily. 

In having this age rated system not only would this help protect the younger end of the community base but also allow for better tagging from a viewer standpoint. Some viewers may not want to see swearing or a lot of blood and gore in their streams, but equally may not know of the game they are checking out, this could go some way to help combat that. 

(Game ratings exist for a reason, whether we choose to follow them or not as a streaming platform, they inform on content and the recommend rating of someone who should play, yes there are people mature enough to play such games at a younger age, but still there is reason for the ratings.)

It would merely be a way to inform people out there of the type of content they are likely to be viewing. It would also help protect at the younger end of our userbase. I also feel it would allow streamers to help better indiciate what type of content they are putting out there. The teen rating wouldn't need to be heavily enforced like the 18+ rating, it would merely be there to help people make a better informed choice about what they are about to view. 

Let me know your thoughts!

**Comment by Paco1342** [comment]
Would the rating for a stream be determined solely by game content, or by total stream content? For example if a streamer is playing a game that is rated for all ages, but cursing a lot - is this all ages? is it Teen? Where do we draw the line?

**Reply by Ribenaberri** [reply]
I feel it should be down to the streamer, however, we could reccomend if theres going to be a lot of swearing then too label as Teen. I think it should be content defined for sure, but equally if someone isn't labelled as it and is swearing, a warning perhaps just to inform them hey maybe this would be better in here. 

I mean for sure its not up too us to police peoples kids, but at the same time, we want to help protect people where possible, and also help people make the most informed choice about their content (afterall, that is what the tagging system is for) and i feel this would go a great way towards that. 

I feel between the GCT we could come up with a working strategy to help inform people how best to use these tags.

**Reply by ChalkLive** [reply]
I think this drawing of the line is where we are going to run into the most challenges to be honest, because it leaves a huge grey area of "Oh that person is maybe swearing more than I am so they should be 18+ but I only need to be Teen". It just leaves the door open for a huge amount of inconsistency.

If we state that some pretty clear guidelines on what NEEDS to flagged at 18+ (with the age verification), and then allow streamers to make up their own mind on what on top of that they WANT to flag, does this leave us in any issues legally? 

I think at some point we need to look at the trade off between ensuring consistency and helping people make informed decisions about the kind of content they are likely to be seeing.

**Reply by Ribenaberri** [reply]
I think whatever the upshot, if the 3 tier thing doesn't come in. I'm glad we are having this conversation and I hope that perhaps a little more towards clarity of what people may see in the platform/ a little part for streamers to consider when flagging maybe ? 

But honestly, I'm really glad this has been a thing and thanks for all.the responses so.far !

**Comment by Cykotiq** [comment]
Stream rating and game rating are not the same and one cannot be used to represent the other - a streamer can be playing a game with violent scenes and just not play the violent parts of the game on stream. Likewise a streamer can play a kid friendly game like roblox and be swearing throughout it.

I feel like with the COPPA laws, and the requirement to be 13+ or supervised by and adult at all times, it would be downright immoral for us to encourage the use of what is essentially a "kid friendly" rating.

The 18+ rating is there for a legal reason laid out in the TOS regarding the few exceptions we make for nudity (not-fully-clothedness). Adding a proper "E for everyone" rating to streams would go directly against what COPPA requires from us, which is PG-13 (parental guidance for anyone under 13). It is not our job to police/supervise/protect other people's kids since they shouldn't be on the site alone in the first place. Making a safe space for them only gives parents a false sense of security - adding more risk to the equation when unsupervised children come across something they shouldn't be looking at.

regarding Mixer, they called it family friendly, implying it could be safely watched "as a family" - ie. the parents/guardians should still be there. I also found the unofficial policy of "don't ask/don't tell" (your age) to be really messed up as it encourages streamers to hide behind willful ignorance, so they (both streamer and mixer) have plausible deniability in the event there is an incident involving a child.

if streamers want to tag their streams as  "family friendly" then fine - that is not so much a proper stream/audience rating, as it is a description of its content - which is what tags were made for.

**Reply by ScottKemosobe** [reply]
Forgot to say it in my post, but about the who COPPA thing, I agree. I don't see much of a reason to have more than the standard and "18+" ratings because people are not allowed to have an account unless they are 13+ anyway.

**Comment by ScottKemosobe** [comment]
I would assume the rating would also be determined by the game as well, like you say CoD would be 16+, but CoD is a M rated game (in the US). And although M is only 17+, everyone basically treats it as 18+ as most shops require you to even be 18 to buy them. And that factor also goes into the rules of "Games containing brief nudity" as those are also only M. I feel like if people are going to play M rated games, or are streamers who "swear like a sailor" no matter what game they are playing, or do anything that basically makes them be "M rated", they should rate the stream as "18+".

In relation to just swearing, at least for US standards, Teen rated stuff only gets minor amounts of swearing. I think PG-13 rated movies are only afforded less than 10 swear words, and according to info I could find, only allowed a single "F-bomb". Going off a rating system wiki for swearing, "The ESRB allows for mild language is appropriate for an E-rated and E10+-rated game. Moderate language is okay to the T rating, while strong language is an M rating, with only a few notable exceptions." And "Moderate Language" I think is stuff like "damn" and possibly some minor swear words.

I do personally feel the rating of the game should reflect the rating of the stream, otherwise it will be impossible for the GCT to actively police every single stream playing an M rated game but not marking themselves as a M rated stream and there will then be people slipping through the cracks until someone else reports them.

**Reply by Ribenaberri** [reply]
As it stands right now, if its M rating, but has no nudity in it, it would be classed as being able to be streamed under the anything category, as defined by the letter of the TOS. Hence trying to think of a way to make it a bit safer for everyone. 

UK side ESRB M equates to 16+ in most cases, not 18+.

**Reply by ScottKemosobe** [reply]
I can't find anything that says CoD is rating PEGI 16 as everything I find says it is PEGI 18.

But either way. Could just simplify it to 2 different labels. The standard/normal one, which probably doesn't need a specific tag, and one for "Mature" and anyone streaming an M/PEGI 16+ rated game (or whatever else standards are used) has to use the "Mature" tag. And the "Mature" tag, or whatever it ends up being called, can trigger an automatic warning when you enter the channel that the viewer has to click saying they are okay with it, like with Twitch.

Regardless of how it ends up being, I do still feel the game should determine if the stream is rated mature or not, otherwise streamers will 100% be not tagging it to try and be seen more and it will cause a LOT of needless policing for the GCT. If they are only looking for a tag, it makes it a lot easier than having to actively watch a stream just to see if they are doing the M rated stuff.

**Reply by Ribenaberri** [reply]
I was looking at CoD: MW remastered, as an example: the new Medal of Honor and Peaky Blinders are 2 other examples. But either way I think we are along the same thinking in terms of approach, we at least agree there needs to be more than their is now.  

While I don't agree a game age rating should be a measure of the stream directly, I do feel the content overall should be. (For example, Destiny 2 is 16, but I feel theres no reason why, with the right streamer, they should have to be a Mature tagged streamer, same for Spider-Man remastered on PS5). However, as we stand at the moment a "mature" stream would be 18+ which is locked behind an age verification. 

Which, in itself, age verification for streams that need it, absolutely 100%. 

But that grey area of 16+ (Wether by context of game content, or context of streaming content i.e. a lot of swearing), shouldn't be amongst the general streaming populus, not without at least some warning or tag or something. But, as it stands right now: 

If you follow the letter of the TOS: 

If you are a sleep streamer, a body painter or there is partial nudity in your game, you must tag as M/18+ and thus be subject to age verification. 

Anyone else, is all the same. Theres no framework, no community guidelines to help advise where you should place yourself nothing. Which I think from a GCT perspective creates a whole host of issues. Whereas if theres at least some guidelines and framework in place, we can be like hey you, do you think maybe youd be better in this age tag because: heres reasons why. 

Perhaps my initial proposal seemed heavy handed, and it is, but outside of anyone else pitching ideas, I posed mine. As this is definitely something I feel needs to be discussed.

**Reply by ScottKemosobe** [reply]
Regardless, the game rating should be a factor in the rating for the stream, otherwise it invites people to abuse the stream rating system by avoiding saying the stream is rated X because it might cause them not to be seen. Just because CoD has an ability to turn off blood and such (from what I have heard) which might possibly bring it down to a T rating doesn't mean everyone playing is doing that. It would require the GCT to go into every single person's stream streaming those games to see if they are playing the mature content. And this might not be that hard at start, but also need to think future.

And in relation to game ratings tying in to stream rating, it is also optics. If a bunch of people come in and see no one streaming CoD or GTA under mature, it will make them feel they are okay to do so as well. And when they get punished for it, it will come off entirely like unfair treatment. Like I have heard a story of a father playing GTA with his like 3 year old or something, but he follows road laws and taught colors and such with it. So technically, he could stream under family friendly. But allowing even that 1 case just invites everyone else to for potentially more viewers and also invites scrutinization from everyone for allowing someone to stream GTA as family friendly.

EDIT: And we would also need to have every country's rating under 1 thing, otherwise it screws over some people. Like if Escape from Tarkov is PEGI 16 in the UK, they get off better than the US where we would have to stream under 17+. And if there are some CoDs rated PEGI 16, those specific ones also get off better as all CoDs since 4 have been rated M (CoD 1-3 were rated T).

**Comment by Ribenaberri** [comment]
Im not basing it on game ratings I am merely using game ratings as an example, those ratings exist for a reason, as to inform content of the game, not that all people dont play them.

As the rules stand right now literally any stream that isn't sleeping, body art or may have nudity is classed as being safe for anyone on the platform. This includes 14 year olds.

Now to say that 14 year olds should be openly allowed to access all and any content on the platform outside of those 3 situations seems less than thinking about what is best for everyone in my opinion, and a warning system is warranted.

Everyone was merely an example. For all I care you could call them 14+, 16+ and 18+, the naming is merely semantics and not at the heart of what I am trying to put across here.

The point is thus, say I am streaming Escape from Tarkov, this is a 16+ game because it has blood and swearing etc in it. The content of the game is violent, aggressively so in places. There's also themes relating to drugs, in fact the game openly encourages the use of stims in the game as a meta win condition. Many of the streamers who play this also swear a lot, get angry when they lose and swear even more.

Should that content be appropriate for a 14 year old?

At the same time, the game isn't so violent as to warrant an 18+ rating, and swearing isn't enough to warrant an 18+ rating either. Plus a lot of 16 year olds play games such as Tarkov, Call of Duty etc, so too put them in an 18+ category immediately stops people being able to stream that game or will lead to people lying about their age in order too be able to stream that game.

Hence, a middle ground proposal, mixer had it, though in my opinion it was a little too tight on the enforcement.

But i do feel there should be an argument for a middle ground, purely based on the content of the game and the streamer. Especially if you want to allow for protection at the younger end of the bracket.

Just because you aren't LEGALLY required to make a decision, doesn't mean that decision isn't in the best interests on the people on the platform.

Im not saying that they should be age verified, just that streamers should have some system whereby the content is rated appropriately so people can make an informed choice about the content they are watching.

And with reference to someone playing roblox and swearing a lot, well im not sure that would consititue as being appropriate to everyone (everyone being 14+ in this instance) and should have some warning attached i.e. 16+ or something.

**Reply by Cykotiq** [reply]
I should also point out that ESRB rating is only one of a couple dozen or so game rating standards worldwide, and what is considered ok in one country may not be the same in another country - "the sims 4" for example ranges from 6+ to 18+depending on the rating system. Some developers even release modified versions of games for certain countries. [SOURCE](https://www.kaspersky.com/blog/game-ratings/32492/)

> Just because you aren't LEGALLY required to make a decision, doesn't mean that decision isn't in the best interests on the people on the platform.

I would argue the opposite actually. If you are not required to apply a rating to it, why would you risk rating it inappropriately and risk a class action lawsuit or massive fines? rating a stream higher than it should be will never be a issue, but rating a stream lower than it should be ABSOLUTELY could be a problem. You are also creating extra work for the GCT because now they need to check up on "immature/kid friendly" streams too. If it gets them better viewership, streamers are very likely to rate them "immature" even if they wouldn't normally - i remember this being the case on mixer as well

**Reply by Ribenaberri** [reply]
Yes, but my point is as it stands right now, unless a game has nudity. You are risking issues. Because all games are open to all ages on the platform to access. Theres literally no way right now to rate a stream higher than it should be, at all. Theres no scope for that within the TOS. But equally to state games such as Tarkov and Cod should be 18+ streams would instantly cut out a huge section of the community from streaming those games. So there should be a middle ground in there somewhere. 

If you have an alternative approach im all for it, this was more to highlight the issue with anyone of any age on the platform being able to access all content outside of nudity, body art, and sleeping. Because the TOS only affects gameplay if it has nudity.

**Reply by Ribenaberri** [reply]
Unless we go for Mature rated, and then age verification required, as 2 separate things? 

So Mature rated could encompass all that grey area which currently has noowhere to live. Which would allow for a stricter approach to the TOS in regards to content which would be seen as more adult. 

And for those bits in the TOS which are absolutely, categorically 18+, have an option for age verification (which clone was talking about implementing to those tagged 18+)

**Comment by Ribenaberri** [comment]
I should point out here. This is absolutely only one idea. And more of a starting point for a discussion. 

Another option would be a knowledge base article about things to consider when age tagging your stream or something. 

I just feel we need to at least do something to try and help cover that grey area a bit.  :)

**Comment by artofcodgtv** [comment]
As someone who streams a main game which is classified as ESRB Mature, which is equivalent to PEGI 16, I feel I couldnt stream within the open to all category, I swear, and the game has characters that swear within it. However my community has people who are 16/17 years old, so streaming 18+ is a No if your requiring age verification from people. 

This would alienate viewers as I feel I shouldnt be streaming in a sort of "this area is for under 18's" but for my community to be able to watch me I would need to, unless you want there to be loads of under 18's with fake accounts set up to be able to watch people. Without a middle ground I feel you are going to be signing up to more issues. 

I agree its not the platforms issue to moderate what peoples children are/arent playing, but as a platform you do have a moral view to do as much as possible to protect people. And allowing people to stream rated M/16+ games in an open setting with swearing without some sort of tagging to state this contains Mature content then I feel your open to many many issues. 

There has to be some sort of definition between Family Friendly, and 18+, or you will alienate a lot of people who may have wished to move over as they will feel that there is no place for them. As because without a more PG/16+/Mature content area there would be no place for my streams, as my content is not 18+, and I would not commit to using a platform without a place for my community to exist.

**Reply by Ribenaberri** [reply]
You'd still be able to optionally tag yourself as mature. The primary concern is how many would do so, unless made too or strongly urged too, because that could effect potential viewership.

**Reply by artofcodgtv** [reply]
I feel that it should definitely be a Mandatory tag to select either family friendly, mature, or 18+. This clearly defines that as a streamer you have set a clear boundary as to what your content will contain. 

It would be easy enough to for people to report people who are mis-tagging and for appropriate action to be taken.

**Reply by Cykotiq** [reply]
Unless we are going to implement some sort of lockout/age-gating this whole 3 rank system is pointless since it'd serve no better purpose than regular tags

**Reply by Ribenaberri** [reply]
If you have to set it before going live then you have to acknowledge those ratings exist. If its just a regular tag there's no need or reason to include it, and many could choose to just not.

**Reply by Cykotiq** [reply]
> If you have to set it before going live then you have to acknowledge those ratings exist. If it's just a regular tag there's no need or reason to include it, and many could choose to just not.

I was actually referring to the viewer end of the equation. If "teen" streams still show up in search results (for viewers who do not wish to see them), or those viewers are not shown a "verify your date of birth" or "are you sure you wish to view this content?" type prompt before viewing, these are no more useful than an extra [vanilla] tag on the stream. Either way "family friendly" streamers need to explicitly set the tag/flag and nobody else will.

Twitch has only an 18+ flag and they do just fine without a "family friendly" stream rating. If streamers want young audiences in their stream they can add their own family friendly tags and those viewers can search for that. I don't think everyone else should be pushed to cater to an immature audience who the platform is not intended for, just so they don't lose viewers - this is the difference between an official rating system, and user-generated tags - one is a restriction, the other is an advertisement.

I really believe user-generated tags can be a game-changer and I think we should give them a chance to prove themselves before adding any official/mandatory tags (except those required by the TOS)

**Reply by Ribenaberri** [reply]
Twitch have a mature tag, not an 18+ tag. A tag which isn't locked behind age verification, merely a yes I understand this is mature tick box. A lot of 16+ streamers see Mature as 16+ and as such choose to stream within that because twitch has such strong TOS and a better safe that sorry approach is adopted. Our TOS however is simple, sleeping, nudity, body art. Everyone outside of that right now can do as they please and we as the GCT can't stop them, there's no recourse, no rules against that. 

However, if you put that Mature tag behind an age verification (which are present, the only thing in place is such) it becomes something which is entirely unnecessary for those who fit within mature but aren't 18+  such as artofcod in the above example. So where do they go? Which is precisely why I'm having this conversation. 


Whilst I understand what you're saying about not catering to immature audiences, the simple fact is this. By allowing users age 14 years old and above in the platform, but very definition that system is seen suitable and fit for all 14 year old. Saying you must be at least this mature, doesn't work, the rules state 14+, therefore the platform is deemed safe for everyone within that too use. 

The way I see it. There's 2 systems at play. Mature, and age verification required. But they can't be the same system. It just wouldn't work. But I also feel having no recourse for someone to stream without an.age related tag, leads to an exposing users, an exposure which should be minimised.

How we do that, if you have another idea, I'm open to listening. But sit and see, shouldn't be our approach imo. As a community driven company, protecting that community even if its only a small portion of that community should be considered important because protecting all of the community is literally part of the GCT job.

**Reply by ChalkLive** [reply]
I think you've hit the nail on the head when you say the 2 systems in place. Mature (tagged) and age verification required (flagged). For me, provided you aren't meeting one of the requirements in ToS to be flagged (age verification required), which could be expanded to include 18+ games (but thats another discussion), then its up to the creator to determine if they want to use a Mature tag or not.

I know a lot of creators that use the mature tag on Twitch, not because of any sort of age verification but because they want to demonstrate the kind of content they are likely to encounter there. I don't see any reason why it couldn't work in a similar manner here.

Yes there will be times where someone who is probably making mature content (but within the ToS guidelines for flagging), doesn't use a Mature tag. Is it our job to police that if they aren't breaking any ToS rules? And if so then we have to be drawing the line on some very grey areas.

**Comment by Ribenaberri** [comment]
Okay so I have an **adjusted propsal** based of the feedback recieved thus far. 

We implement 2 tags M and M+ (again, naming isnt the important thing right now) 

So we have Mature for those who feel that their content is more aimed towards an older audience, but that audience isn't specifically 18 years of age, nor does it need to be. 

We have Mature + which is verified Mature. Those who, as termed by the TOS can only stream to an auidence of 18 years or older. 

Then everyone else is just there. They can tag as family friendly, or not, whatever. 

We could potentially even make it so, if a streamer tags themselves as M as pop up comes up saying do you require age verification yes/no *with a link to where in the TOS it states what would require it*

We could perhaps as a GCT put together a consideration document to help inform those who are on the platform about what tagging in Mature means, and why its important to consider this tag when making the stream i.e. because of the younger demographic on the platform etc. 

It doesn't need to be overly in depth, but enough to help guide someone into that category.

If a GCT member comes across a stream they feel should be in the M category, we put them to be observed. (Maybe they are constantly playing M rated games, or their content seems to be more M). If the GCT as a whole can agree unanimously that they should, we perhaps fire them an email just to say we feel maybe they would be better to to flag as 'x'. 

We also have a document which explains what these ratings are for everyone else. 

I know there was talking of potentially allowing certain tags to be selected as not seen, so perhaps thats also a consideration. So for example, at the younger end of the group if a parent is presiding over how their 14/15 year old uses the platform, they could set it so M and M+ streams cannot be seen, thus allowing some level of protection. (will those kids turn it off? maybe, but at least that option of protection has been afforded) 

This allows for a better understanding of the system, it affords protection at the younger end, and it allows the GCT better implementation when moderating content. The GCT will be watching streams to make sure they haven't been tagged incorrectly in any case, so this could become part of that process. 

Its a much less abrasive tactic that my initial proposal, and i feel covers a lot of the worries addressed here. 

But again, im open to more feedback to tweak things further :D

**Reply by EmberShadow** [reply]
I feel like this would, without a doubt, prevent that disenfranchisement that some streamers and also viewers would feel if we merely had that 18+ toggle which I just don't think is good enough.

This way makes things just that much more inclusive to our community and give a lot of folks who are under the age of 18 much more freedom to stream or watch the content that they actually want rather than locking them behind an age wall.

**Reply by ChalkLive** [reply]
I think thats probably the best solution. However we end up working it or naming it, it allows us to age restrict content that HAS to be age restricted via ToS, as well as allowing creators the freedom to mark their content as mature but not have it be age restricted. 

It also opens up the reverse effect where creators can mark their channels as Family Friendly if they truly are catering to that audience.

Of course the tags can still be a reason for reporting them if they are being misused, but thats the same idea as someone streaming an unknown game under Fortnite because they want to try and get more views. We've already discussed giving ourselves the ability to change that as we see fit.

**Comment by Heiwa1580** [comment]
So I have some time to write.

There are basically a few things I think we need to limit to minors out of concern for [Obscenity laws.](https://www.justice.gov/criminal-ceos/citizens-guide-us-federal-law-obscenity) 

To start I think we shouldn't allow anyone under the 18 to do sleeping streams, body art that requires undressing, and games that have nudity in them. I think that these styles of streams need to have some kind of filtration to them. Simply put minors should not be doing streams where someone can watch them sleep, do a body painting stream where they are essentially undressed, and doing streams that have nudity. That being said I don't think we have to have an 18+ tag per se on the first two. But I think we need an age restriction on who can do those style streams. (Writing this out now the 18+ may have nothing to do with this whole idea but in the TOS it is written that way)

Minors are defined as anyone under the [age of majority](https://www.law.cornell.edu/wex/minor#:~:text=All%20states%20define%20an%20"age,guardian%20unless%20they%20are%20emancipated.) and it is almost universally accepted as 18.

There are commonly 3 different disctinctions of ages when it comes to laws like this. Child, Minor, Juvenile. These are general definitions and can be used interchangeably but for general purposes they usually fall like this. Child is anyone 13 and under, Minor is anyone 14 to Age of Majority, and Juvenile is anyone typically between 14 to 17. But any of these three do not have the same responsibilities as adults and are who we need to make sure to be mindful of with decisions like this. 

Those labelled as Child are not allowed to make an account on the platform. It doesn't mean their parents can't make them a profile but that account is there parents. We cannot give the appearance that we have content suitable for Children otherwise we will be at risk for [COPPA](https://www.ftc.gov/enforcement/rules/rulemaking-regulatory-reform-proceedings/childrens-online-privacy-protection-rule) laws. We have clear TOS that say we do not have accounts for Children but if we have settings that say Child Friendly or anything in relation to Children we run the risk of FTC looking into it. Our platforms default rating would be Teen or PG-13

Those who are minors we have to be aware of giving access to content that could be seen as pornographic. Many games like the Witcher, Cyberpunk, Red Dead Redemption can have elements of pornography in them. These I think should be behind some kind of age rating simply because of the nude elements. 

Beyond that though I don't believe that we have any legal obligations that would be harmful to us. But to Berri's point I think that we can set a standard that would allow a better control for parents (who have allowed their 14-17 year old kid) to control what they see. I think also that there will be a point in time when the FCC starts looking into streaming and may even force ratings. 

All that being said I think that it would be difficult to enforce a full rating system based on content. I think it would be hard enough to constantly check even for just the three limits we have (sleeping, nudity, and undressed body art). But I think as long as any system we had setup took those certain laws and rules in mind I think we are fine. 

My suggestion would be an optional slider that can be set per channel or per stream. There would be general guidelines of fitting into a specific rating but ratings are not required. I think that TOS could be written to make it work.

Unrated (UR) - No rating set no guidelines for content

Family Friendly (FF) - No swearing, no crude jokes, no harsh language, no games over T or equal. Suitable for a family to watch together.

Mature Teen (MT) - Swearing is fine, crude jokes, harsh language, no nudity, no sleeping streams, . This setting would not have a before you enter warning.

Mature (M) - Swearing, crude jokes, harsh language, no nudity,  This setting would have a mature content warning but no age verification

Mature 18 (M18) - Swearing, crude jokes, games with nudity, This setting would have age verification.

I don't think this would be a perfect system by any means but I think it would be better to standardize this type of thing then to let it be up to chance if we were to add something like this

**Reply by Heiwa1580** [reply]
All this said I think all we need for Alpha is just one of the last 2 things.
